src/Controller/SiteController.php

```
@@ -68,8 +68,9 @@ public function signupSubmitAction(Application $app, Request $request)
 
         $repo = $app->getUserRepository();
         try {
-            $user = $repo->register($username, $email);
+            $user = $repo->register($app, $username, $email);
         } catch (Exception $e) {
+            var_dump($e);exit;
```

Can you return a proper error response object?